### PR TITLE
[cloudflare-turnstile] updated render parameters types as per cloudfl…

### PR DESCRIPTION
**Pull request by artem328**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `pnpm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

If changing an existing definition:

- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://developers.cloudflare.com/turnstile/get-started/client-side-rendering/#configurations
- [x] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the `package.json`.

----
Not sure about version bump since this package defines types for in-browser loaded SDK and there is no publicly available source code for it

### CODE REVIEW

**Review by lbarthon — APPROVED** [review]

**Comment by lbarthon** [comment]
Would you mind doing a patch bump? Types version doesn't match the product version

**Comment by artem328** [comment]
@lbarthon I would bump the version, but what is the product's version? I could not find it anywhere publicly

**Comment by lbarthon** [comment]
@artem328 there is no public version, `0.2.0` sgtm

File: types/cloudflare-turnstile/package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
     "private": true,
     "name": "@types/cloudflare-turnstile",
-    "version": "0.1.9999",
+    "version": "0.2.9999",
```
Comment (lbarthon) [inline types/cloudflare-turnstile/package.json:4]: why `.9999`? `0.2.0` sgtm?

Comment (artem328) [reply types/cloudflare-turnstile/package.json:4]: This is a [requirement](https://github.com/DefinitelyTyped/DefinitelyTyped?tab=readme-ov-file#how-do-definitely-typed-package-versions-relate-to-versions-of-the-corresponding-library) of the DefinitelyTyped repo

> Note that the version in package.json should only contain major.minor version (e.g. 10.12) followed by .9999

Comment (lbarthon) [reply types/cloudflare-turnstile/package.json:4]: TIL

**Review by lbarthon — APPROVED** [review]

**Comment by lbarthon** [comment]
Ready to merge

**Pull request merged**